### PR TITLE
chore(workflow): remove dependabot workflow

### PR DESCRIPTION
**Pull request by AugustinMauroy**

### CODE REVIEW

**Review by ovflowd — APPROVED** [review]
LGTM! Let's remove Dependabot for now.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-version: 2
-updates:
-  - package-ecosystem: github-actions
-    directory: '/'
-    schedule:
-      interval: weekly
-    open-pull-requests-limit: 10
-  - package-ecosystem: npm
```
Comment (ovflowd) [inline .github/dependabot.yml:8]: Can we only remove the `npm` one and keep the `github-actions` one?

Comment (AugustinMauroy) [reply .github/dependabot.yml:8]: Shure. We can also put evrythings on comment.

**Review by ghost — COMMENTED** [review]
LTGM!

**Review by ghost — DISMISSED** [review]

**Review by mikeesto — APPROVED** [review]
For context: I think the reason we are temporarily disabling Dependabot is because of the significant migration work happening on the `/major/website-redesign` branch, and trying to reduce the likelihood of dependency conflicts when it lands. A decrease in notifications within our GitHub inboxes is an added bonus too...

File: .github/dependabot.yml (hunk 1 above)
Comment (ovflowd) [reply .github/dependabot.yml:8]: I'd rather just remove. We can always add back by checking git history.

Comment (AugustinMauroy) [reply .github/dependabot.yml:8]: Ok, lets go to remove.

**Comment by ovflowd** [comment]
@MaledongGit dependabot alerts are about the security alerts feature. Not the PRs for dependency updatee.

**Review by aymen94 — APPROVED** [review]

**Comment by ghost** [comment]
> @MaledongGit dependabot alerts are about the security alerts feature. Not the PRs for dependency updatee.

@ovflowd：

I submitted a PR to make a feature request to the dependabot, however the author of GitHub's waiting for my response, and I don't have the authentication to verify whether it works? Can any of you check it? I was told not only security alerts but updates as well……

Refs：https://github.com/dependabot/dependabot-core/issues/7072

If this still doesn't work, I'll submit and give him a reason why, otherwises I'll close this feature.

**Comment by ovflowd** [comment]
@MaledongGit dependabot security alerts are an entirely different feature from dependabot automatic dependency updates, as I mentioned before.

Edit: It seems that we're able indeed to disable it via options, but I'm unable to find that option. Will give it another try.

File: .github/dependabot.yml (hunk 1 above)
Comment (ovflowd) [reply .github/dependabot.yml:8]: @AugustinMauroy I'm still waiting.

**Review by ovflowd — CHANGES_REQUESTED** [review]
Still waiting for the unrelated changes to be undone 👀 (referring to remove the github-actions config for dependabot)

**Comment by ghost** [comment]
> @MaledongGit dependabot security alerts are an entirely different feature from dependabot automatic dependency updates, as I mentioned before.
> 
> Edit: It seems that we're able indeed to disable it via options, but I'm unable to find that option. Will give it another try.

Yes, It would be better if we can keep the file :)

**Comment by ovflowd** [comment]
I'm closing this PR as there's a few controversial things happening at play. I'll reach out GitHub support.

**Comment by ghost** [comment]
Just a tip only (no need to change any more but just let you what to do if we meet such problems): 

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#disabling-dependabot-version-updates

Just a very simple sample XD XD XD

**Pull request closed without merging**